build-system/tasks/assets.js

```diff
@@ -21,20 +21,27 @@ const jsifyCssAsync = require('./jsify-css').jsifyCssAsync;
 const pathLib = require('path');
 const {endBuildStep, mkdirSync} = require('./helpers');
 
-function assets() {
+async function assets() {
+  // Make directories.
   mkdirSync('dist');
-  fs.copySync('assets/loader.svg', 'dist/loader.svg', {overwrite: true});
-  return Promise.all([
+  mkdirSync('dist/i18n');
+
+  // Compile CSS.
+  await Promise.all([
     compileCss('assets/swg-button.css', 'dist/swg-button.css', {
       sourceMap: false,
     }),
     compileCss('assets/swg-mini-prompt.css', 'dist/swg-mini-prompt.css', {
       sourceMap: false,
     }),
-  ]).then(() => {
-    mkdirSync('dist/i18n');
-    fs.copySync('assets/i18n/', 'dist/i18n/', {overwrite: true});
-  });
+  ]);
+
+  // Copy SVGs.
+  fs.copySync('assets/loader.svg', 'dist/loader.svg', {overwrite: true});
+  fs.copySync('assets/i18n/', 'dist/i18n/', {overwrite: true});
+
+  // Remove extra files.
```

Comment:
Yep, expanded!

Reply:
Related to my other comment, it would be preferable if we didn't put unneeded files in `assets/` to begin with. We can leave that as a future improvement though.